hwilib/commands.py

```diff
@@ -269,6 +269,9 @@ def process_commands(args):
       descriptor = Descriptor.parse(args.desc, args.testnet)
       args.fingerprint = descriptor.fingerprint
 
+    if hasattr(args, "psbt") and args.psbt:
+      args.fingerprint = args.fingerprint
```

Comment:
... Why? What's the point of this?

Reply:
Leftover from an earlier version, will remove.